Virtusize/Tests/APIRequestTests.swift

```diff
@@ -157,10 +157,11 @@ class APIRequestTests: XCTestCase {
         XCTAssertNotNil(apiRequest?.httpBody)
         let actualParams = try? JSONDecoder().decode(VirtusizeGetSizeParams.self, from: apiRequest!.httpBody!)
         XCTAssertNotNil(actualParams)
-        XCTAssertEqual(actualParams?.additionalInfo.count, 5)
-        XCTAssertEqual(actualParams?.additionalInfo["gender"]?.value as? String, "male")
+        XCTAssertEqual(actualParams?.items.first?.additionalInfo.count, 4)
+       
+        XCTAssertEqual(actualParams?.items.first?.additionalInfo["gender"]?.value as? String, "female")
```

Comment:
Could we have enum for gender ?

```swift
.FEMALE /// female
.MALE /// male
```

Reply:
this the value which will be given by user,  in framework we have enum , but this is from user side test case

Reply:
Understood thanks